### PR TITLE
Issue 275  Correct array order in python scripts

### PR DESCRIPTION
**Pull request by timspainNERSC**
# Correct array order in python scripts
## Fixes \#275

The array-ordering changes to the python scripts that generate the netCDF restart files were missed out of the array-ordering PR #373. They are now included here.

### CODE REVIEW

File: run/era5_topaz4_maker.py [hunk 1]
```diff
@@ -199,9 +199,9 @@ def rotate_velocities(u, v, angle):
...
     node_lat = node_coords[:, :, 1]
-    nx = node_lon.shape[0] - 1
-    ny = node_lon.shape[1] - 1
-    element_shape = (nx, ny)
+    ny = node_lon.shape[0] - 1
+    nx = node_lon.shape[1] - 1
+    element_shape = (ny, nx)
     element_lon = np.zeros(element_shape)
     element_lat = np.zeros(element_shape)
     # interpolate lon and lat from nodes to elements, to leave nx x ny arrays
```
Comment (MarionBWeinzierl) [inline run/era5_topaz4_maker.py:207]:
```suggestion
    # interpolate lon and lat from nodes to elements, to leave ny x nx arrays
```

File: run/era5_topaz4_maker.py [hunk 2]
```diff
@@ -199,9 +199,9 @@ def rotate_velocities(u, v, angle):
     # assume lon and lat are 0 and 1 coords
     node_lon = node_coords[:, :, 0]
     node_lat = node_coords[:, :, 1]
-    nx = node_lon.shape[0] - 1
-    ny = node_lon.shape[1] - 1
-    element_shape = (nx, ny)
+    ny = node_lon.shape[0] - 1
```
[the remark this answers is not in the document]
Comment (timspainNERSC) [reply run/era5_topaz4_maker.py:202]: Yes, just changing which of the source data dimensions we label as x and which as y. We want the array points in the final file to be in the same order as in the source file, so the same dimension varies fastest. But here we change the name of that fastest dimension.

File: run/make_init25kmNH.py [hunk 3]
```diff
@@ -4,7 +4,11 @@
 import time
 import math
 
+<<<<<<< issue275_3_arrayorderwithavengeance
```
[the remark this answers is not in the document]
Comment (timspainNERSC) [reply run/make_init25kmNH.py:7]: Oops.

File: run/era5_topaz4_maker.py (hunk 1 above)
Comment (timspainNERSC) [reply run/era5_topaz4_maker.py]: Done

**Review by MarionBWeinzierl — APPROVED** [review]

**Pull request merged**